core/src/main/java/com/linecorp/armeria/common/logging/LoggingDecoratorBuilder.java

```diff
@@ -36,6 +36,15 @@
     private LogLevel requestLogLevel = LogLevel.TRACE;
     private LogLevel successfulResponseLogLevel = LogLevel.TRACE;
     private LogLevel failedResponseLogLevel = LogLevel.WARN;
+    private Function<? super RequestLog, LogLevel> requestLogLevelMapper =
+            log -> requestLogLevel();
+    private Function<? super RequestLog, LogLevel> responseLogLevelMapper =
+            log -> log.responseCause() == null ? successfulResponseLogLevel() : failedResponseLogLevel();
+    private boolean isSetRequestLogLevel;
```

Comment:
nit: how about `isRequestLogLevelSet`?

Reply:
OK. Let me fix it.

Reply:
Fixed

Reply:
Anyway. Just curious. IsSetXXX is used in the thrift generated code. What is more proper?

Reply:
`is...Set` sounds more natural to me.

Reply:
Yeah. The naming is too hard to me. 😃